prerequisites/check-version.js

```diff
@@ -0,0 +1,28 @@
+var semver = require("semver");
+var requiredVersion = require("../package.json").engines.node;
+
+function requiredToList(requiredVersion) {
+	return requiredVersion.split("||").filter(Boolean).map(function(version) {
+		return new semver.Range(version).set[0];
+	}).map(function(set) {
+		if(!set) {
```

Comment:
`set` is an array, so it won't ever be falsey, but if we use the above, there's the possibility it's value will be empty

```js
if (!!set[0].value) {
  return null;
}
```